lib/matplotlib/contour.py

```diff
@@ -249,9 +249,9 @@ def get_label_width(self, lev, fmt, fsize):
                         .get_text_width_height_descent(lev, fsize))
         elif ismath:
             if not hasattr(self, '_mathtext_parser'):
-                self._mathtext_parser = mathtext.MathTextParser('bitmap')
-            img, _ = self._mathtext_parser.parse(lev, dpi=72,
-                                                 prop=self.labelFontProps)
+                self._mathtext_parser = mathtext.MathTextParser('agg')
```

Comment:
this specific use is going away via https://github.com/matplotlib/matplotlib/pull/16171.